harness/determined/deploy/aws/aws.py

```diff
@@ -317,11 +353,30 @@ def deploy_stack(
     if stack_exists(stack_name, boto3_session):
         print("True - Updating Stack")
 
-        update_stack(stack_name, template_body, boto3_session, parameters)
+        if not no_prompt:
+            tags = get_tags(stack_name, boto3_session)
+            prompt_needed = False
+            if constants.deployment_types.TYPE_TAG_KEY not in tags:
+                print("Previous value of --deployment-type is not known.")
```

Comment:
It's worth a longer explanation here, like maybe:

"Previous value of --deployment-type is not known.  Versions of determined as late as 0.17.2 did not annotate deployed clusters, and it was the responsibility of the user to make updates with the same --deployment-type.  Note that if you are sure you did not set --deployment-type before, your cluster would have deployed as --deployment-type simple (the default)."

In general, any time you can show some long detailed explanation to users _exactly when they need it_ (rather than in a man page or docs page), that's good bang for your buck.  The more precise you are here, the fewer users will contact us confused in OSS slack.

Reply:
I'm on board with that. Manually wrapping at 80 chars. We don't do much of that elsewhere, but on most modern terminals it's unlikely to be very noticeable. This would be noticeable, IMO.